recipe/meta.yaml

```diff
@@ -51,6 +68,8 @@ outputs:
       run:
         - python
         - {{ pin_subpackage("pybind11-global", exact=True) }}
+      run_constrained:
+        - pybind11-abi =={{ abi_version }}
```

Comment:
Would it make sense to pin pybind11-global too, just in case someone tries to use it that directly? It is valid to do so, you'd just get the "global" locations of headers and CMake files, and no Python package.

Reply:
Ahh ok. Yes. I'll add run_constrained to that package too. I did not know that.

Reply:
done!